libcontainer/cgroups/stats.go

```diff
@@ -20,6 +20,12 @@ type CpuUsage struct {
 	// Total CPU time consumed per core.
 	// Units: nanoseconds.
 	PercpuUsage []uint64 `json:"percpu_usage,omitempty"`
+	// CPU time consumed per core in kernel mode
+	// Units: nanoseconds.
+	PercpuUsageInKernelmode []uint64 `json:"percpu_usage_in_kernelmode"`
```

Comment:
I would use shorter yet still descriptive names, e.g. `KernelPerCpu` and `UserPerCpu`.

Reply:
I prefer the current`PercpuUsageInKernelmode` for consistency with `PercpuUsage` and `UsageInKernelmode`

Reply:
As @AkihiroSuda noticed, I decided to use these names to keep consistency.